### DISCUSSION TITLE
Input Field Error messages

### DISCUSSION THREAD

**Discussion by JordanMed** [solspace/craft-freeform#253] [Questions & Answers] [answered]
Is there a way to translate "This field is required" error I can't seem to figure how to do it. I thought it would be similar to event.options.successBannerMessage = 'My custom success message'; or event.target.freeform.setOption("successBannerMessage", "This is a custom success message.");

I found the answer Please ignore this

**Comment by kjmartens** [comment, marked as answer]
Glad you were able to find the solution. For others with this question, these are translated with static translation files: 🙂 
https://docs.solspace.com/craft/freeform/v3/setup/translating.html
